modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -71,7 +72,7 @@ You can then reassign the appropriate external client provider for that API.
 ** New APIs use the new external IdP.
 * After you assign an external client provider to an API, you cannot return to using the default native Anypoint Platform client provider.
 +
-You can user either the default IdP or one or more external IdPs. You cannot use the default provider and an external client provider together.
+You can use either the default IdP or one or more external IdPs. You cannot use the default provider and an external client provider together. To change an API's IdP from an external provider to an Anypoint Platform client provider, remove every IdP from the environment in which the API belongs.
```

Comment:
I suggest that you GLOBALLY search the material in these topics and choose to refer to one but not both of the following after initially introducing the two terminologies:

the default IdP or one or more external IdPs
the default native client [identity] provider or one or more external client [identity] providers

In one of my previous editing lives, we recommended that once you provide an abbreviation, and it is more commonly used than the full spelling, then you should use only it in following text (in that topic, anyway). So maybe you want to use just "default IdP" and "external IdP"?

Reply:
Based on the first sentence, I would omit the second: "either" should make clear that you cannot use both.

Reply:
Clarify how the third sentence relates to the previous two?
Also...WHY or under what circumstances would the audience want or need to change the IdP?